frontend/src/products/Hint/HintComponent.jsx

```
@@ -0,0 +1,52 @@
+            style={{
+              backgroundColor: headerBackgroundColor,
+              color: headerColor,
+            }}
+          >
+            <h3>{header}</h3>
+          </div>
+        )}
+        <div
+          className="preview-content"
+          style={{ color: textColor }}
+          dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(content) }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)